server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -319,18 +321,21 @@ private QuestionDefinition insertTextQuestionDefinition() {
         .getResult();
   }
 
-  private ProgramDefinition insertProgramWithBlocks(String name, QuestionDefinition nameQuestion) {
+  private ProgramDefinition insertProgramWithBlocks(
+      String adminName, String displayName, QuestionDefinition nameQuestion) {
     try {
-      ProgramDefinition programDefinition =
-          programService
-              .createProgramDefinition(
-                  name,
-                  "desc",
-                  name,
-                  "display description",
-                  "https://github.com/seattle-uat/civiform",
-                  DisplayMode.PUBLIC.getValue())
-              .getResult();
+      ErrorAnd<ProgramDefinition, CiviFormError> programDefinitionResult =
+          programService.createProgramDefinition(
+              adminName,
+              "desc",
+              displayName,
+              "display description",
+              "https://github.com/seattle-uat/civiform",
+              DisplayMode.PUBLIC.getValue());
+      if (programDefinitionResult.isError()) {
+        throw new Exception(programDefinitionResult.getErrors().toString());
+      }
```

Comment:
Why this addition?

Reply:
The main reason was for debuggability when I had a bug in the implementation. Previously, when an error was returned, we were unconditionally calling `ErrorAnd.getResult`, which threw an unhelpful exception of "No result provided." rather than the actually useful error indicating the `CiviFormError` from the downstream call.